### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by mwellick**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,32 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self,
+                 name: str,
+                 health: int = 100,
+                 hidden:
+                 bool = False) -> None:
```
Comment (EdAlekseiev) [inline app/main.py:8]:
```suggestion
                 hidden: bool = False) -> None:
```

File: app/main.py [hunk 2]
```diff
@@ -1 +1,32 @@
...
+class Herbivore(Animal):
+    def hide(self) -> None:
+        self.hidden = not self.hidden
+
+
+class Carnivore(Animal):
+    @staticmethod
+    def bite(herbivore_animal: Herbivore) -> None:
+        if isinstance(herbivore_animal, Herbivore) \
+                and not herbivore_animal.hidden:
```
Comment (EdAlekseiev) [inline app/main.py:29]: Use `()` instead of `\`

Comment (EdAlekseiev) [inline app/main.py:29]:
```suggestion
        if (isinstance(herbivore_animal, Herbivore) 
                and not herbivore_animal.hidden):
```

**Review by EdAlekseiev — CHANGES_REQUESTED** [review]

**Review by olena-hul — APPROVED** [review]

**Comment by olena-hul** [comment]
Good job!

**Pull request closed without merging**